compiler/il/OMRILOpCodesEnum.hpp

```diff
@@ -53,13 +53,32 @@
    bload,    // load byte
    sload,    // load short integer
    lload,    // load long integer
+
+   //Read barrier is used to represent loads with side effects like check for GC, debugging etc.
+   //It is the same as the corresponding load except that it needs to be anchored under a
+   //treetop. The children and symbol of a read barrier are the same as the corresponding load.
```

Comment:
@0xdaryl do we have any template for how to properly document IL? IMO we should add some structure to the documentation.